packages/material-renderers/src/complex/MaterialTableControl.tsx

```diff
@@ -65,6 +65,7 @@ import NoBorderTableCell from './NoBorderTableCell';
 import TableToolbar from './TableToolbar';
 import { ErrorObject } from 'ajv';
 import merge from 'lodash/merge';
+import { ArrayTranslations } from '@jsonforms/core/lib/i18n/arrayTranslations';
```

Comment:
This should be imported directly from the jsonforms/core package. If that is not possible, please export the new definitions appropriately.
```suggestion
import { ArrayTranslations } from '@jsonforms/core';
```